### DISCUSSION TITLE
End of August devenv sprint title

### DISCUSSION THREAD

**Discussion by j8xixo12** [sciwork/swportal#74] [General]
I just finished the web page of end of August devenv sprint, but I still haven't figured out the title of sprint....

The first coming into my mind is package management, but I know this topic is very difficul :P

Dose anyone have some idea?!

**Comment by yungyuc** [comment]
If determining the subject needs time, how about just calling it "August sprint" or "late August sprint"?

**Comment by yungyuc** [comment]
@j8xixo12 filed a PR #75 for discussions.

**Comment by tai271828** [comment]
In my opinion, `sciwork sprint` is good enough if there is no specific topic or goal to achieve in that sprint event.

All projects relate to sciwork and devenv could be the items running in the sprint event. The following come to my mind:

- Build an application other than `solvcon`
- The swportal aforementioned in https://github.com/sciwork/swportal/pull/75
- The usual routine: working on open issues of swportal https://github.com/sciwork/swportal/issues and devenv https://github.com/solvcon/devenv/issues
- Countless projects of computational science
